app/src/main/res/values/styles.xml

```diff
@@ -468,6 +468,28 @@
     <item name="android:textAlignment">viewStart</item>
   </style>
 
+  <style name="AddMultiProfileHeading" parent="TextView.Common1">
```

Comment:
Please not add new style `AddMultiProfileHeading` instead of `AddMultiProfileHeading` use `Heading5` and add `android:textColor="@color/component_color_shared_secondary_4_text_color"` in `TextView`

Avoid using colors from `color_def` use color from `component_color_shared`